src/components/Questions/Body.vue

```diff
@@ -298,9 +321,25 @@ export default defineComponent({
     }
 
     function preventKeypressIfApplicable(event: KeyboardEvent) {
-      // checks if character limit is reached in case it is set
-      if (!hasCharLimit.value) return;
-      if (!charactersLeft.value) event.preventDefault();
+      if (isQuestionTypeSubjective.value) {
+        // checks if character limit is reached in case it is set
+        if (!hasCharLimit.value) return;
+        if (!charactersLeft.value) event.preventDefault();
```

Comment:
Maybe I'm not completely right about the logic, but aren't the above two statements the same? 
If you've hit the limit, it means there are no characters left, no?

Reply:
this is not part of my PR but I will take a look at this